backend/open_api_spec.yml

```diff
@@ -0,0 +1,790 @@
+        - collection_visibility
+        - revised_at
+        - is_primary_data
+        - X_normalization
+        - dataset_assets
+        - tombstone
+        - assay
+        - organism
+        - tissue
+      properties:
+        id:
+          $ref: "#/components/schemas/dataset_uuid"
+        curator_tag:
+          $ref: "#/components/schemas/curator_tag"
+        name:
+          type: string
+        revision:
+          type: integer
+        collection_id:
+          $ref: "#/components/schemas/collection_uuid"
+        collection_visibility:
+          $ref: "#/components/schemas/visibility"
+        revised_at:
+          type: number
+        is_primary_data:
+          $ref: "#/components/schemas/is_primary_data"
+        X_normalization:
+          type: string
+        dataset_assets:
+          type: array
+          items:
+            $ref: "#/components/schemas/dataset_asset"
+        tombstone:
+          type: boolean
+        assay:
+          $ref: "#/components/schemas/ontology_element_array"
+        organism:
+          $ref: "#/components/schemas/ontology_element_array"
+        tissue:
+          $ref: "#/components/schemas/ontology_element_array"
+        disease:
+          $ref: "#/components/schemas/ontology_element_array"
+        sex:
+          $ref: "#/components/schemas/ontology_element_array"
+        ethnicity:
+          $ref: "#/components/schemas/ontology_element_array"
+        development_stage:
+          $ref: "#/components/schemas/ontology_element_array"
+        dataset_deployments:
+          type: array
+          items:
+            type: object
+            properties:
+              url:
+                type: string
+        linked_genesets:
+          type: array
+          items:
+            type: string
+        cell_type:
+          $ref: "#/components/schemas/ontology_element_array"
+        cell_count:
+          type: integer
+        X_approximate_distribution:
+          $ref: "#/components/schemas/distribution"
+        schema_version:
+          type: string
+    dataset_preview:
+      type: object
+      required:
+        - id
+        - curator_tag
+        - tissue
+        - assay
+        - disease
+        - organism
+        - tombstone
+      properties:
+        id:
+          $ref: "#/components/schemas/dataset_uuid"
+        curator_tag:
+          $ref: "#/components/schemas/curator_tag"
+        tissue:
+          $ref: "#/components/schemas/ontology_element_array"
+        assay:
+          $ref: "#/components/schemas/ontology_element_array"
+        disease:
+          $ref: "#/components/schemas/ontology_element_array"
+        organism:
+          $ref: "#/components/schemas/ontology_element_array"
+        tombstone:
+          type: boolean
+    dataset_asset:
+      type: object
+      properties:
+        id:
+          type: string
+        dataset_id:
+          type: string
+        filetype:
+          type: string
+          enum: [H5AD, RDS, CXG]
```

Comment:
@jahilton - would you validate that the CXG asset was created through the API or use the UI to explore the dataset to validate?

Reply:
And this is a case where the permanent link is actually in datasets_deployment.

Reply:
@brianraymor can you please clarify? Jason will need to validate whether the the asset was created via an API submission vs UI submission?

Reply:
I do not see that the CXG asset in `dataset_assets` is valuable to curators, so I would suggest that it is removed:

"dataset_assets": [
          {
            "created_at": 1638224578.183704,
            "dataset_id": "524e045e-e74c-4e00-9884-d5c3bef3d862",
            "filename": "explorer_cxg",
            "filetype": "CXG",
            "id": "e8b0e0ce-41aa-4666-ae2c-c10098c98cc6",
            "s3_uri": "s3://hosted-cellxgene-prod/524e045e-e74c-4e00-9884-d5c3bef3d862.cxg/",
            "type": "REMIX",
            "updated_at": 1638224578.183711,
            "user_submitted": true
          },

On the other hand, the permanent link in `dataset_deployments` can be both used to validate that the CXG was created (from the API) and to capture the link to share with data submitters if needed:

        "dataset_deployments": [
          {
            "url": "https://cellxgene.cziscience.com/e/524e045e-e74c-4e00-9884-d5c3bef3d862.cxg/"
          }
        ],

Reply:
Ok. Keep permanent link to cxg but remove from dataset_assets (leaving just H5AD and RDS).

Reply:
>would you validate that the CXG asset was created through the API or use the UI to explore the dataset to validate?

I would use the API.
>I do not see that the CXG asset in dataset_assets is valuable to curators, so I would suggest that it is removed:

I agree

Reply:
removed CXG enum (`3b8cc6a4`)